infrastructure/application/services/hasura.ts

```diff
@@ -172,8 +168,8 @@ export const createHasuraService = async ({
 
   // TODO: bump awsx to 1.x to use the FargateService scaleConfig option to replace more verbose config below
   const hasuraScalingTarget = new aws.appautoscaling.Target("hasura-scaling-target", {
-    // start conservative, can always bump max as required
-    maxCapacity: 3,
+    // maxCapacity should consider compute power of the RDS instance which Hasura relies on
+    maxCapacity: parseInt(config.require("hasura-planx-api-key")),
```

Comment:
When I ran tried to push Hasura on staging to breaking point (i.e. hit it with a workload well beyond our satisfactory target), AWS responded by spinning up a 3rd server, which failed to start because the database couldn’t handle any more connections (Hasura servers maintain 20 or so connections even when they have no active queries).

For reference, this saturation point was ~200 connections for the `db.t3.small` instance we've provisioned on staging. There's no reason to put it under that strain.

By comparison, the prod RDS runs on `db.t3.medium`, which is twice as powerful.

Reply:
Note that if "connection churn" ever becomes a real problem, we can put something like [PgBouncer](https://www.pgbouncer.org/) in front of it to do connection pool management (or AWS’s solution, [RDS Proxy](https://aws.amazon.com/rds/proxy/)).

Reply:
Oops typo here I think - I think you mean to 
```
maxCapacity: parseInt(config.require("hasura-scaling-maximum"))
```

Reply:
@jessicamcinchak oops thank you, nice catch!